### ISSUE TITLE
Submit package on packagist

### ISSUE THREAD

**Issue by chewett** [novus/nvd3#1285]
Now we have a composer file the final step is to register it on packagist.org

You need to login to the site under an account, submit the url of the github repository and it is done.

This would bring this project in line with the d3 repository and make it exceptionally easy to get the libraries.

Linked: https://github.com/novus/nvd3/pull/1270

**Comment by liquidpele** [comment]
https://packagist.org/packages/novus/nvd3

It doesn't seem to automatically get the tags for versions though.

**Comment by chewett** [comment]
It will automatically pull the tags but only once those tagged versions have a composer.json file.

So until either the tags have a composer.json merge into a branch and the tags are updated it wont work. However once 1.8.2 is released with a composer.json it should work fine.

**Comment by liquidpele** [comment]
okay.
